modules/ttdBidAdapter.js

```diff
@@ -413,9 +404,10 @@ export const spec = {
     let topLevel = {
       id: bidderRequest.auctionId,
       imp: validBidRequests.map(bidRequest => getImpression(bidRequest)),
+      app: firstPartyData.app || {},
       site: getSite(bidderRequest, firstPartyData),
       device: getDevice(),
-      user: getUser(bidderRequest),
+      user: getUser(bidderRequest, firstPartyData),
```

Comment:
why not take fpd on device?

Reply:
thanks for pointing device and pmp out, we don't expect to need them right now but its good to pass those along

Reply:
Okay